### ISSUE TITLE
crashes when clicking on versions button

### ISSUE THREAD

**Issue by bbrendon** [ytti/oxidized-web#187]
I saw some other people this similar error but I can't get past it. Running the latest version from docker hub.  It's basically a fresh install.

ArgumentError at /node/version
wrong number of arguments (given 0, expected 1+)
file: versions.haml location: count line: 24

```
---
username: xxx
password: xxx
model: junos
resolve_dns: true
interval: 3600
use_syslog: false
debug: false
threads: 30
timeout: 20
retries: 3
prompt: !ruby/regexp /^([\w.@-]+[#>]\s?)$/
#rest: 127.0.0.1:8888
rest: 0.0.0.0:8888
next_adds_job: false
vars: {}
groups: {}
models: {}
pid: "/root/.config/oxidized/pid"
crash:
  directory: "/root/.config/oxidized/crashes"
  hostnames: false
stats:
  history_size: 10
input:
  default: ssh, telnet
  debug: false
  ssh:
    secure: false
  ftp:
    passive: true
  utf8_encoded: true

# output:
#   default: file
#   file:
#     directory: "/root/.config/oxidized/configs"
output:
  default: git
  git:
    user: IT
    email: it@redacted.com
    repo: "/root/.config/oxidized/deviceconfigs.git"

source:
  default: csv
  csv:
    file: "/root/.config/oxidized/router.db"
    delimiter: !ruby/regexp /:/
    map:
      name: 0
      model: 1
    gpg: false
model_map:
  juniper: junos
  cisco: ios

```

```
Mar 15 13:09:23 redacted.com docker[10925]: *** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
Mar 15 13:09:23 redacted.com docker[10925]: *** Running /etc/my_init.d/10_syslog-ng.init...
Mar 15 13:09:24 redacted.com docker[10925]: Mar 15 20:09:24 4520b1a958b7 syslog-ng[13]: syslog-ng starting up; version='3.13.2'
Mar 15 13:09:25 redacted.com docker[10925]: *** Booting runit daemon...
Mar 15 13:09:25 redacted.com docker[10925]: *** Runit started as PID 22
Mar 15 13:09:25 redacted.com docker[10925]: Mar 15 20:09:25 4520b1a958b7 cron[31]: (CRON) INFO (pidfile fd = 3)
Mar 15 13:09:25 redacted.com docker[10925]: Mar 15 20:09:25 4520b1a958b7 cron[31]: (CRON) INFO (Running @reboot jobs)
Mar 15 13:09:25 redacted.com docker[10925]: I, [2019-03-15T20:09:25.847901 #28]  INFO -- : Oxidized starting, running as pid 28
Mar 15 13:09:25 redacted.com docker[10925]: I, [2019-03-15T20:09:25.849572 #28]  INFO -- : lib/oxidized/nodes.rb: Loading nodes
Mar 15 13:09:26 redacted.com docker[10925]: I, [2019-03-15T20:09:26.141864 #28]  INFO -- : lib/oxidized/nodes.rb: Loaded 3 nodes
Mar 15 13:09:27 redacted.com docker[10925]: Puma starting in single mode...
Mar 15 13:09:27 redacted.com docker[10925]: * Version 3.11.4 (ruby 2.5.1-p57), codename: Love Song
Mar 15 13:09:27 redacted.com docker[10925]: * Min threads: 0, max threads: 16
Mar 15 13:09:27 redacted.com docker[10925]: * Environment: development
Mar 15 13:09:27 redacted.com docker[10925]: * Listening on tcp://0.0.0.0:8888
Mar 15 13:09:27 redacted.com docker[10925]: Use Ctrl-C to stop



Mar 15 13:15:12 redacted.com docker[10925]: 2019-03-15 20:15:12 - ArgumentError - wrong number of arguments (given 0, expected 1+):
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/oxidized-web-0.13.1/lib/oxidized/web/views/versions.haml:24:in `count'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/oxidized-web-0.13.1/lib/oxidized/web/views/versions.haml:24:in `block in singleton class'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/oxidized-web-0.13.1/lib/oxidized/web/views/versions.haml:-2:in `instance_eval'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/oxidized-web-0.13.1/lib/oxidized/web/views/versions.haml:-2:in `singleton class'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/oxidized-web-0.13.1/lib/oxidized/web/views/versions.haml:-5:in `__tilt_47231683954460'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/tilt-2.0.9/lib/tilt/template.rb:170:in `call'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/tilt-2.0.9/lib/tilt/template.rb:170:in `evaluate'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/tilt-2.0.9/lib/tilt/haml.rb:24:in `evaluate'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/tilt-2.0.9/lib/tilt/template.rb:109:in `render'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:823:in `render'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:677:in `haml'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/oxidized-web-0.13.1/lib/oxidized/web/webapp.rb:241:in `out'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/oxidized-web-0.13.1/lib/oxidized/web/webapp.rb:165:in `block in <class:WebApp>'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:1611:in `call'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:1611:in `block in compile!'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:975:in `block (3 levels) in route!'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:994:in `route_eval'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:975:in `block (2 levels) in route!'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:1015:in `block in process_route'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:1013:in `catch'
Mar 15 13:15:12 redacted.com docker[10925]: /var/lib/gems/2.5.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:1013:in `process_route'
...
```

**Comment by dnyt84** [comment]
same problem.
Did you ever solve this?

**Comment by ytti** [comment]
Duplicate of ytti/oxidized#2049

**Comment by didawubo1** [comment]
I also encountered the same problem, did you solve it？

**Comment by wk** [comment]
This is solved in 0.28.0
